src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -83,6 +83,120 @@ public static class DnnCatalog
             return new DnnRetrainEstimator(env, options, DnnUtils.LoadDnnModel(env, modelPath, true));
         }
 
+        /// <summary>
+        /// The options for the <see cref="ImageClassificationTransformer"/>.
+        /// </summary>
+        public sealed class Options
```

Comment:
We don't need 2 "Options" classes. We only need one.

This appears to be a duplicate of the `ImageClassificationEstimator.Options` class. One of these should be removed.

Reply:
What should happen is the nested `ImageClassificationEstimator.Options` class should become public, and then this one can be deleted.

---
In reply to: [337563694](https://github.com/dotnet/machinelearning/pull/4362#discussion_r337563694) [](ancestors = 337563694)

Reply:
The two classes have most members in common, except that DnnCatalog.Options has featuresColumnName, and  ImageClassificationEstimator.Options() takes in InputColumns and OutputColumns. Using ImageClassificationEstimator.Options directly would change the necessary arguments that ImageClassification() would take. I can make that change if that is acceptable. 


---
In reply to: [337568602](https://github.com/dotnet/machinelearning/pull/4362#discussion_r337568602) [](ancestors = 337568602,337563694)